data/strings/strings.txt

```diff
@@ -14313,52 +14313,6 @@
     zh-Hans = 选择菜肴
     zh-Hant = 選擇料理
 
-  [username]
```

Comment:
This one should be fixed by adding this commit https://github.com/organicmaps/organicmaps/commit/094e6e729a0de6252b08925383167528d735c34a